src/buffer.h

```diff
@@ -150,6 +150,11 @@ class Buffer {
   /// initial count. This requires the format to have been set.
   void ResizeTo(uint32_t element_count);
 
+  /// Resizes the buffer to hold |element_count| elements. This requires the
```

Comment:
Update comment to match as this doesn't work on element_count.

Reply:
I updated the comment, let know if you'd like to add anything specific